content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,185 @@
+single source of truth for our model management.
+
+To make use of our model registry, we will need some trained models first. To
+speed things up, we will start from a git
+[repository](https://github.com/iterative/example-get-started-model-management)
+with a model training pipeline already set up and ready to use.
+
+To see how you can track experiments and set up training pipelines with DVC and
+DVCLive, have a look at our getting started guide for
+[experiment management](/doc/start/experiments).
+
+## DVC Model registry overview
+
+In DVC Studio we can access the model registry by clicking on Models in the top
+menu. This will show you a dashboard with all models from all projects you have
+access to. You can check out our
+[public model registry example](https://studio.iterative.ai/team/Iterative/models).
+
+From the dashboard you will have an overview of all models, latest model
+versions as well stages each of the model versions is assigned to. You can also
+see which git repository for each model and get more details for it by clicking
+on the model name.
+
+Here you will see some extra information about a particular model - a
+description of the model, any labels that were assigned and particularly the
+history of all model registry actions on that selected model. For each model
+version you can also have a look at its metrics tracked by the experiment
+tracked.
+
+## Adding models
+
+Let's now train a model and add it to the model registry.
+
+We have three options how to add a model to the model registry. In this guide,
+we will be using DVCLive and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to save the
+model and add it to the model registry. Open the training notebook
+`notebooks/TrainSegModel.ipynb` in our example repository and in the last cell
+of the notebook add the method call inside the `with Live(...)` statement as
+follows.
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+If we now run the code and commit the result to git (and push it to our git
+remote), the new model will show up in the model registry in Studio. You should
+see something like the following picture.
+
+![Newly added model in the Model Registry](/img/mr-newly-added-model.png)
+
+<details id="push-click-to-see-other-ways-to-add-models">
```

Comment:
WDYT about a details section showing the `dvc.yaml` output of the `live.log_artifact()` call?

Reply:
Yes, that is a good point, now it is perhaps too much magic and details sections do not slow down any people who don't care about them (or just want to go through the guide quickly on a first pass)

Reply:
Hey! Hard to understand this last comment, as it seems Inconsistent user-story understanding: The guide is too long for a "first pass".Maybe as a user I experience this page differently ?. As a user, when I want to go faster, I would go directly to the [demo ](https://studio.iterative.ai/team/Iterative/models), and maybe the starter page skip this. If this works aims to [substitute ](https://dvc.org/doc/studio/model-registry) existing page, please don´t hesitate to include in the PR description so contributors can get some context.  

I would add the details in a section that could be extended , or at the end. 

I might be experiencing model registry differently , don´t hesitate to clarify here. Thanks!